lambdas/session/src/main/java/uk/gov/di/ipv/cri/address/api/handler/SessionHandler.java

```diff
@@ -20,6 +25,20 @@ public APIGatewayProxyResponseEvent handleRequest(
                 new APIGatewayProxyResponseEvent();
         apiGatewayProxyResponseEvent.setStatusCode(HttpStatus.SC_CREATED);
         UUID uuid = UUID.randomUUID();
+
+        try {
+            AmazonDynamoDB client = AmazonDynamoDBClientBuilder.standard().build();
```

Comment:
You want to specify the region read somewhere it helps with performance see.   https://github.com/aws-samples/serverless-patterns/blob/main/apigw-lambda-dynamodb-sam-java/src/main/java/com/example/utils/DDBUtils.java#L17

So region would be `EU_WEST_2`

I couldn't find the story related to this I suppose it is similar to our story here maybe not as elaborate https://govukverify.atlassian.net/browse/KBV-90

You could follow the `lib/persistence/dataStore` pattern

![image](https://user-images.githubusercontent.com/3749690/154736484-d2618f22-5b3f-422e-b8fd-b4fcc5229106.png)

The ipv-core used this pattern and we've used it as well, I think it would work here as well

Reply:
https://github.com/alphagov/di-ipv-cri-kbv-api/blob/main/lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/persistence/DataStore.java

Reply:
Thanks for explaining this @jkunle 👍